src/e2e/step-definitions/external-systems.ts

```diff
@@ -95,7 +99,9 @@ When(
     );
     await expect(caseNumberLocator).toHaveCount(2);
 
-    const wizardResultDiv = smartDocumentsWizardPage.locator("#wizard-result");
+    const wizardResultDiv = smartDocumentsWizardPage.locator(
+      '[role="status"][aria-live="polite"]',
```

Comment:
This locator is less obvious, a comment above the constant could give some details on why this has to be selected on `aria-live` property value `polite`.